docs/introduction.rst

```diff
@@ -0,0 +1,111 @@
+Introduction
+============
+
+The XAITK-Saliency toolkit implements a class of XAI algorithms known
+as `saliency algorithms`. The basic application pipeline is shown in Figure 1:
+
+.. figure:: figures/intro-fig-01.png
+
+*Figure 1: A basic AI pipeline.*
+
+In this scenario, an AI algorithm operates on an input (text, image,
+etc.) to produce some sort of output. Saliency algorithms build on
+this as shown in Figure 2:
```

Comment:
Linking to the idea of saliency maps introduced in the next paragraph:

"to produce visual explanations in the form of saliency maps as shown in Figure 2 ".

Reply:
sounds good